core-primitives/sgx-runtime-primitives/src/types.rs

```diff
@@ -0,0 +1,39 @@
+use sp_runtime::{
```

Comment:
Can you add a file header here?

Reply:
Added it in b1374ecce5541b9e799cca11b9b63de2bc581b30.